python/plugins/db_manager/dlg_sql_window.py

```diff
@@ -653,7 +653,9 @@ def _getSqlQuery(self):
         return sql
 
     def _getExecutableSqlQuery(self):
-        sql = self._getSqlQuery()
+        sql = self._getSqlQuery().strip()
```

Comment:
```suggestion
        sql = self._getSqlQuery().strip(' \n\r;')
```

Would that help when there's a trailing space/newline?

e.g.

```
select *
from x;

```

or do the check on the uncommented sql (which removes newlines)

Reply:
I'm fixing another issue here. Please file another ticket.

Reply:
My proposal aimed at a more generic approach for *exactly* this issue and nothing else.
I'll open a follow up pull request.

Reply:
Ok, thank you!